libraries/stdlib/src/kotlin/collections/Maps.kt

```diff
@@ -52,37 +55,65 @@ public fun <K, V> linkedMapOf(vararg values: Pair<K, V>): LinkedHashMap<K, V> {
     return answer
 }
 
-/** Returns the [[Map]] if its not null otherwise it returns the empty [[Map]] */
+/**
+ * Returns the [Map] if its not null, or the empty [Map] otherwise.
+ */
 public fun <K,V> Map<K,V>?.orEmpty() : Map<K,V>
        = if (this != null) this else emptyMap()
 
+/**
+ * Checks if the map contains the given key. This method allows to use the `x in map` syntax for checking
+ * whether an object is contained in the map.
+ */
 public fun <K,V> Map<K,V>.contains(key : K) : Boolean = containsKey(key)
 
-/** Returns the key of the entry */
+/**
+ * Allows to access the key of a map entry as a property. Equivalent to `getKey()`.
+ */
 public val <K, V> Map.Entry<K, V>.key: K
     get() = getKey()
 
-/** Returns the value of the entry */
+/**
+ * Allows to access the value of a map entry as a property. Equivalent to `getValue()`.
+ */
 public val <K, V> Map.Entry<K, V>.value: V
     get() = getValue()
 
-/** Returns the key of the entry */
+/**
+ * Returns the key of the map entry. This method allows to use multi-declarations when working with maps,
+ * for example:
+ * ```
+ * for ((key, value) in map) {
+ *     // do something with the key and the value
+ * }
+ * ```
+ */
 public fun <K, V> Map.Entry<K, V>.component1(): K {
     return getKey()
 }
 
-/** Returns the value of the entry */
+/**
+ * Returns the value of the map entry. This method allows to use multi-declarations when working with maps,
+ * for example:
+ * ```
+ * for ((key, value) in map) {
+ *     // do something with the key and the value
+ * }
+ * ```
+ */
 public fun <K, V> Map.Entry<K, V>.component2(): V {
     return getValue()
 }
 
-/** Converts entry to Pair with key being first component and value being second */
+/**
+ * Converts entry to [Pair] with key being first component and value being second
```

Comment:
This doc uses first/second *component*, while doc for mapOf/hashMapOf/… uses first/second *value*. Should use the same word.